resources/sass/global/_variables.scss

```diff
@@ -23,3 +23,4 @@ $yellow: #ffed4a;
 $green: #38c172;
 $teal: #4dc0b5;
 $cyan: #6cb2eb;
+$i-blue: #2699FB;
```

Comment:
i don't love this variable name but it's fine

Reply:
lol me either but we need to override bootstraps built-in variables

Reply:
It will change anyways on STR-53